srcpkgs/kamoso/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'kamoso'
+pkgname=kamoso
+version=21.04.3
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="
```

Comment:
Add qt5-host-tools kcoreaddons kconfig for cross build.

Reply:
I have added `qt5-host-tools kcoreaddons kconfig`